src/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/azure/azure-sdk-actions
+
+go 1.18
+
+require github.com/stretchr/testify v1.7.1
+
+require (
+	github.com/davecgh/go-spew v1.1.0 // indirect
```

Comment:
How confident are we with taking dependencies on these third-party packages?

Reply:
This is only a test dependency and will not be included in current method of action execution (`go run`). When we start publishing a release build of the tool, we can add stuff to make sure these test dependencies aren't included in it.